lib/ansible/modules/cloud/azure/azure_rm_aci.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_aci
+version_added: "2.4"
+short_description: Manage an Azure Container Instance (ACS).
+description:
+    - Create, update and delete an Azure Container Instance.
+
+options:
+    resource_group:
+        description:
+            - Name of a resource group where the Container Services exists or will be created.
+        required: true
+    group_name:
+        description:
+            - Name of the container Group
+        required: true
```

Comment:
`group_name` can be optional. Azure-CLI even cannot configure it

Reply:
indeed, only API requires group name, so i think we can just do: name + "-group"